core/privc/privc_context.h

```diff
@@ -13,24 +13,28 @@
 // limitations under the License.
 #pragma once
 
-#include <algorithm>
 #include <algorithm>
 #include <memory>
 
 #include "core/paddlefl_mpc/mpc_protocol/abstract_context.h"
 #include "core/paddlefl_mpc/mpc_protocol/abstract_network.h"
-#include "prng_utils.h"
+#include "core/privc3/prng_utils.h"
 
-namespace aby3 {
+namespace privc {
 
 using AbstractNetwork = paddle::mpc::AbstractNetwork;
 using AbstractContext = paddle::mpc::AbstractContext;
+using block = psi::block;
```

Comment:
can we factor the common crypto code from psi? it's weird to rely on psi in privc.